tests/core/test_validator.py

```diff
@@ -55,3 +55,23 @@
     importer = RasaFileImporter(domain_path=invalid_domain)
     validator = await Validator.from_importer(importer)
     assert not validator.verify_utterances()
+
+
+async def test_verify_there_is_example_repetition_in_intents():
```

Comment:
we should also add a test that all duplicated examples are printed correctly

Reply:
I wrote a test called `test_verify_logging_message_for_repetition_in_intents`. @Ghostvv, could you please check if it is the test that you mentioned?

Reply:
looks good